samples/application/simple-cli-go/pkg/fpcadmin.go

```diff
@@ -42,7 +42,7 @@ func NewAdmin(config *Config) *Admin {
 	//defer sdk.Close()
 
 	orgAdmin := "Admin"
-	orgName := "org1"
```

Comment:
This is a good change. Let's make sure that this does not impact the our existing tutorials.

For instance, we would need to set `CORE_PEER_ORG_NAME=org1` in this example.

https://github.com/hyperledger/fabric-private-chaincode/tree/main/samples/deployment/test-network#how-to-use-simple-cli-go

Another example is this one.
https://github.com/hyperledger/fabric-private-chaincode/blob/main/samples/deployment/fabric-smart-client/the-simple-testing-network/env.sh#L46

Reply:
Solved